tests/astrometry_net_data/photocal/andConfig.py

```diff
@@ -0,0 +1,7 @@
+root.defaultMagColumn = 'r'
+root.starGalaxyColumn = 'star'
+#root.variableColumn = 'variable'
```

Comment:
I'm not exactly sure why we need all these configs.  If we don't need them all, I'd like to remove the unused ones.  I would also like to clean up the commented out code.